docs/guidelines/content-guidelines/05-tabs-toggles.md

```diff
@@ -0,0 +1,238 @@
+
+  - All sets of toggles from the given group must contain the `group` attribute with a distinctive ID of this group. For example, add the `cluster-installation` group to categorize all toggle sets that refer to Kyma installation on a cluster.
+  - Every `<summary>` tag in the set must also contain the `label` attribute. For example, add the `GKE` label to all toggles that describe steps for Kyma installation on a GKE cluster in the `cluster-installation` group.
+
+    > **NOTE:** The label does not have to be the same as the summary. If the label is a combination of two or more words, separate them with the dash (`-`). For example, write `<summary label="install-on-gke">`.
+
+  See the example:
+
+```html
+  <div tabs name="prerequisites" group="cluster-installation">
+    <details>
+    <summary label="gke">
+    GKE
+    </summary>
+    ...
+    </details>
+    <details>
+    <summary label="aks">
+    AKS
+    </summary>
+    ...
+    </details>
+    <details>
+    <summary label="gardener">
+    Gardener
+    </summary>
+    ...
+    </details>
+  </div>
+  ```
+
+### Examples
+
+This section contains exemplary uses of the toggle in Markdown documents.
+
+* See the example of the toggle added as a regular document section:
+
+```html
+<div tabs name="Create a Subscription" group="create-subscription">
+  <details open>
+  <summary label="Kyma Dashboard">
+  Kyma Dashboard
+  </summary>
+
+1. Go to **Namespaces** and select the default Namespace.
+2. Go to **Configuration** > **Subscriptions** and click **Create Subscription+**.
+3. Switch to the **Advanced** tab, and provide the following parameters:
+   - **Subscription name**: `lastorder-sub`
+   - **Config**: `maxInFlightMessages: 5`
+   - **Types**: `order.received.v1`
+   - **Service**: `lastorder` (The sink field will be populated automatically.)
+   - **Type matching:**: `standard`
+   - **Source**: `myapp`
+
+5. Click **Create**.
+6. Wait a few seconds for the Subscription to have status `READY`.
+
+  </details>
+  <details>
+  <summary label="kubectl">
+  kubectl
+  </summary>
+
+Run:
+```bash
+cat <<EOF | kubectl apply -f -
+   apiVersion: eventing.kyma-project.io/v1alpha2
+   kind: Subscription
+   metadata:
+     name: lastorder-sub
+     namespace: default
+   spec:
+     config:
+       maxInFlightMessages: "5"
+     sink: 'http://lastorder.default.svc.cluster.local'
+     source: myapp
+     types:
+       - order.received.v1
+EOF
+```
+
+To check that the Subscription was created and is ready, run:
+```bash
+kubectl get subscriptions lastorder-sub -o=jsonpath="{.status.ready}"
+```
+
+The operation was successful if the returned status says `true`.
+  </details>
+</div>
+```
+
+The code renders on `kyma-project.io` as follows:
+
+![](./assets/toggle-flat-structure.png)
+
+* See the example of the toggle added under a list:
+
+```html
```

Comment:
all lines of the code block need 2 more blank spaces to render as part of the list item

```suggestion
  ```html
```

<img width="475" alt="Screenshot 2023-09-04 at 14 00 22" src="https://github.com/kyma-project/community/assets/76950046/747b7730-f922-49cd-a028-bd4940d56bbf">